Source/relay/TourGuide/Items of the Month/2024/Chest Mimic

```diff
@@ -0,0 +1,26 @@
+//2024
+//Chest Mimic
+RegisterResourceGenerationFunction("IOTMChestMimicGenerateResource");
+void IOTMChestMimicGenerateResource(ChecklistEntry [int] resource_entries)
+{
+	if (!lookupFamiliar("Chest Mimic").familiar_is_usable()) return;
+
+	// Title
+    int famExperienceGain = numeric_modifier("familiar experience") +1;
+	int chestExperience = ($familiar[chest mimic].experience);
+	int famExpNeededForNextEgg = (50 - (chestExperience % 50));
+	int mimicEggsLeft = clampi(11 - get_property_int("_mimicEggsObtained"), 0, 11);
+	string [int] description;
+	string url = "familiar.php";
+	{
+		description.listAppend("Currently have " + HTMLGenerateSpanOfClass(chestExperience, "r_bold") + " experience, currently gain " + HTMLGenerateSpanOfClass(famExperienceGain, "r_bold") + " fam exp per fight.");
+		description.listAppend("Need " + HTMLGenerateSpanOfClass(famExpNeededForNextEgg, "r_bold") + " more famxp for next egg. (" + (50/ famExperienceGain) + " fight(s))");
```

Comment:
Don't we need to make this change for the number of fights to be accurate? If I need 4 exp to next egg and get 2 per fight, it should show 2, but the existing code would say 25.

`ceil` makes sure it gets rounded up if there's a fractional amount left (e.g. I need 5 exp to next egg with 2 per fight, it'll divide to 2.5 but round up to 3). `toFloat` makes sure we get the fractional amount instead of chopping it off with integer division.
```suggestion
		description.listAppend("Need " + HTMLGenerateSpanOfClass(famExpNeededForNextEgg, "r_bold") + " more famxp for next egg. (" + ceil(toFloat(famExpNeededForNextEgg) / famExperienceGain) + " fight(s))");
```